### PR TITLE
Updates for 2.6.1

### PR DESCRIPTION
**Pull request by mimaison**
- Copied docs from 2.6.1 artifacts
- Updated downloads.html to include 2.6.1

### CODE REVIEW

File: 26/documentation.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <!--#include virtual="toc.html" -->
   </div>
 	<div class="right">
-		<!--#include virtual="../includes/_docs_banner.htm" -->
+		<!--//#include virtual="../includes/_docs_banner.htm" -->
```
Comment (bbejeck) [inline 26/documentation.html]: @mimaison you'll want to keep the `<!--#include virtual="../includes/_docs_banner.htm" -->` link format 
cf https://github.com/apache/kafka-site/pull/307

File: 26/documentation.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h3 class="anchor-heading"><a id="introduction" class="anchor-link"></a><a href=
       <h3 class="anchor-heading"><a id="uses" class="anchor-link"></a><a href="#uses">1.2 Use Cases</a></h3>
       <!--#include virtual="uses.html" -->
       <h3 class="anchor-heading"><a id="quickstart" class="anchor-link"></a><a href="#quickstart">1.3 Quick Start</a></h3>
-      <!--#include virtual="quickstart.html" -->
+      <!--#include virtual="quickstart-zookeeper.html" -->
```
Comment (bbejeck) [inline 26/documentation.html]: Also, you'll want to keep the `<!--#include virtual="quickstart.html" -->` links as well

cf https://github.com/apache/kafka-site/pull/313

**Comment by mimaison** [comment]
Thanks @bbejeck, I now remember seeing your mention in #307 but I had completely forgotten about it!
I reverted all the `<!--#include virtual="../includes/_docs_banner.htm" -->` tags accordingly

**Comment by mimaison** [comment]
Artifacts are now in maven: https://repo1.maven.org/maven2/org/apache/kafka/kafka-clients/2.6.1/

File: 26/streams/architecture.html [hunk 3]
```diff
@@ -168,7 +168,7 @@ <h3 class="anchor-heading"><a id="streams_architecture_recovery" class="anchor-l
 <div class="content documentation ">
     <!--#include virtual="../../includes/_nav.htm" -->
     <div class="right">
-        <!--#include virtual="../../includes/_docs_banner.htm" -->
+        <!--/#include virtual="../../includes/_docs_banner.htm" -->
```
Comment (bbejeck) [inline 26/streams/architecture.html]: I think you don't want the `/` character here

File: 26/quickstart-docker.html [hunk 4]
```diff
@@ -0,0 +1,204 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (bbejeck) [inline 26/quickstart-docker.html]: You can delete this `quickstart-docker.html` file, we just need `quickstart.html`

File: 26/quickstart-zookeeper.html [hunk 5]
```diff
@@ -0,0 +1,277 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (bbejeck) [inline 26/quickstart-zookeeper.html]: same as above.

**Review by bbejeck — COMMENTED** [review]
@mimaison just a couple of small changes needed but otherwise this LGTM.  I rendered the PR locally and validated some of pages.

**Comment by mimaison** [comment]
Thanks @bbejeck 
I've pushed an update

**Comment by mimaison** [comment]
@bbejeck @mjsax @vvcephei  Can you take a look? I'd like to release 2.6.1 today

**Review by bbejeck — APPROVED** [review]
I verified by rendering locally and clicked on some of links

**Comment by mimaison** [comment]
Thank you!

**Pull request merged**